Gw2 Launchbuddy/MainWindow.xaml.cs

```diff
@@ -1914,12 +1914,12 @@ private void UpdateServerArgs()
 
         private void Window_LostKeyboardFocus(Object sender, System.Windows.Input.KeyboardFocusChangedEventArgs e)
         {
-            if (cinemamode || rb_cinemavideomode.IsChecked == true) { Cinema_MediaPlayer.Pause(); }
+            if (cinemamode | rb_cinemavideomode.IsChecked == true) { Cinema_MediaPlayer.Pause(); }
             BeginStoryboard(this.FindResource("anim_musicfadeout") as System.Windows.Media.Animation.Storyboard);
         }
         private void Window_GotKeyboardFocus(Object sender, System.Windows.Input.KeyboardFocusChangedEventArgs e)
         {
-            if (cinemamode || rb_cinemavideomode.IsChecked == true) { Cinema_MediaPlayer.Play(); } 
```

Comment:
First is to check if it's in cinema mode, second is to check if video mode is the type running. I think it caused a crash or something when running slideshow mode. I don't work on anything outside of personal use, so sorry about no documentation. I could also just be completely wrong, and be using the wrong operator.